### DISCUSSION TITLE
Secondary header Label

### DISCUSSION THREAD

**Discussion by saloeric** [rappasoft/laravel-livewire-tables#1198] [General]
Hello, when using the filters as 'secondary header' with "hiddenFromMenus()", a label now appears that did not appear before. And this label doesn't look good in this case. Is there any method to ignore this label?

**Comment by lrljoe** [comment]
Can you pop me a screenshot please, and let me know if you're using tailwind or bootstrap?
And the type of filter you're using as a header/footer.

**Comment by saloeric** [comment]
Hello, I'm using Bootstrap 4. I'm using it in the header (TextFilter and SelectFilter).

![table-header](https://github.com/rappasoft/laravel-livewire-tables/assets/14850389/77fac465-0911-4ca8-8d2f-e1b74a92566f)

**Reply by lrljoe** [reply]
Thanks for the example, I'm assuming you're referring to the duplicated column name?

I'll take a quick look, I know I wrote the functionality in there to detect whether the filter instance is in the header or footer, will just need to make sure it doesn't break anything!

**Reply by lrljoe** [reply]
This fix will be in the next release, and is in develop now.

**Reply by saloeric** [reply]
thank you

**Comment by lrljoe** [comment]
This should now be remediated, please re-open if it isn't!
